R/WebApi.R

```diff
@@ -142,3 +142,39 @@ getCdmSources <- function(baseUrl) {
 
   do.call(rbind, sourceDetails)
 }
+
+
+#' Get the WebApi configuration for the WebApi instance.
+#'
+#' @details
+#' Get the WebApi configuration for the WebApi instance.
+#'
+#' @param baseUrl        The base URL for the WebApi instance, for example:
+#'                       "http://server.org:80/WebAPI".
+#' @param sourceKeys     A list of string names for the source keys as in
+#'                       configuration page of Atlas.
+#' @return
+#' A list representing the WebApi configuration for the CDM Schema
+#'
+#' @examples
+#' \dontrun{
+#' #
+#'
+#' getConfiguration(baseUrl = "http://server.org:80/WebAPI", sourceKeys = c('source1', 'source2'))
+#' }
+#'
+#' @export
+getConfiguration <- function(baseUrl,sourceKeys){
+  require(dplyr)
```

Comment:
Do not use `require` or `library` in a function. Instead, rely on the double-colon notation to call functions in an imported package, e.g.
```r
dplyr::filter()
```
Otherwise your function will have unintended side-effects, in this case loading the dplyr package in the user's search path, which could lead to problems later on.

Also, make sure the dependency is in the import section of the DESCRIPTION file. I think you just deleted dplyr there?

Reply:
Thank you! I must have substituted dplyr with tidyverse. This probably is not best practice anyways. I have put dplyr back.